content/concept-depositauth.md

```diff
@@ -0,0 +1,59 @@
+# Deposit Authorization
+
+_(Requires the [DepositAuth amendment](reference-amendments.html#depositauth).)_
+
+Deposit Authorization is an optional feature of an [account](concept-accounts.html) in the XRP Ledger. With Deposit Authorization enabled, transactions cannot send value of any kind to the account unless the sender of those transactions is the account itself. This includes transfers of XRP and issued currencies.
+
+## Background
+
+Financial services regulations and licenses may require that a business or entity must know the sender of all transactions it receives. This presents a challenge on a decentralized system like the XRP Ledger where participants are identified by pseudonyms which can be freely generated and the default behavior is for any address to be able to pay any other.
+
+The Deposit Authorization flag introduces an option for those using the XRP Ledger to become compliant with such regulations without changing the fundamental nature of the decentralized ledger. With Deposit Authorization enabled, an account can only receive funds it explicitly approves by sending a transaction. The owner of an account using Deposit Authorization can perform the due diligence necessary to identify the sender of any funds _before_ sending the transaction that causes it to receive the money.
```

Comment:
> The last phrase tripped me up. Does this mean the account sends a transaction to approve or to receive the funds?

Both. The transaction sends an account to receive the funds, which is how it states that it approves of receiving those funds.

Reply:
The exact wording is as it is because there are edge cases where the account's prior transactions provide the "approval" to receive the funds later (as in the case of rippilng and offers).